### ISSUE TITLE
`guild runs comment -l` doesn't list comments

### ISSUE THREAD

**Issue by uduse** [guildai/guildai#380]
`guild runs comment -l` is supposed to have the same behavior as `guild runs comment --list` according to the `--help`:

```
Options:
  -l, --list                List comments for specified runs.
```

However, `-l` shows:
```
guild: option -l is deprecated and was removed in version 0.8 - use -Fl instead
```

While `--list` shows:
```
[1] @e9e378064bec 2022-04-19 09:22:26

  Test comment.
```

My `guild check`:
```
guild_version:             0.8.0
guild_install_location:    /root/.local/share/virtualenvs/moozi-g1CZ00E9/lib/python3.8/site-packages/guild
guild_home:                /root/.local/share/virtualenvs/moozi-g1CZ00E9/.guild
guild_resource_cache:      /root/.local/share/virtualenvs/moozi-g1CZ00E9/.guild/cache/resources
installed_plugins:         config_flags, cpu, dask, disk, dvc, exec_script, gpu, ipynb, keras, memory, perf, python_script, queue, skopt
python_version:            3.8.12 (default, Apr 14 2022, 23:11:54) [GCC 7.5.0]
python_exe:                /root/.local/share/virtualenvs/moozi-g1CZ00E9/bin/python
platform:                  Linux 4.15.0-118-generic x86_64
psutil_version:            5.9.0
tensorboard_version:       2.8.0
cuda_version:              11.4.152
nvidia_smi_version:        450.51.06
latest_guild_version:      0.8.0
```

**Comment by msarahan** [comment]
That's a mistake in the help. It should reflect `-Fl` instead of `-l`

**Comment by msarahan** [comment]
scratch that, that won't work. There's something more annoying here. Thanks for reporting. We'll fix it ASAP.

**Comment by gar1t** [comment]
This is fixed in 0.8.1, which is due out this week! Sorry for the breakage!

**Comment by msarahan** [comment]
0.8.1 has been released. If this continues to be an issue, please re-open this issue or open a new issue.
